### ISSUE TITLE
Email: Intopia Feedback - WCAG 3.0 FPWD (3 - Normative)

### ISSUE THREAD

**Issue by jspellman** [w3c/silver#468]
[Comment from Email](https://lists.w3.org/Archives/Public/public-agwg-comments/2021Feb/0015.html): 

Outcomes are normative. Should guidelines, methods, critical errors, and outcome ratings be normative or informative, and why?

Guidelines are normative; they are used to address functional needs on specific topics and be directly tested against.

Critical errors can be both normative and informative. When critical errors specify an issue that will stop a user from participating or completing a process, critical errors are normative in that they are absolute. However, when critical errors are part of a fail rating system that describes how badly one has passed or failed and how to improve the error's criticality, this may be considered informative.

Methods are informative; inclusive of detailed information and guidance on achieving a particular outcome. Outcome ratings can be both normative and informative. Like critical errors, outcome ratings can guide how to improve the outcome while maintaining an informative aspect of strictly intended outcomes.

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
